src/com/palmergames/bukkit/towny/listeners/TownyTypeListener.java

```diff
@@ -0,0 +1,67 @@
+package com.palmergames.bukkit.towny.listeners;
+
+import com.palmergames.bukkit.towny.TownyAPI;
+import com.palmergames.bukkit.towny.event.actions.*;
+import com.palmergames.bukkit.towny.event.damage.TownyDamageEvent;
+import com.palmergames.bukkit.towny.object.TownBlock;
+import org.bukkit.event.EventHandler;
+import org.bukkit.event.Listener;
+
+/**
+ * Listener for the internal Towny Custom Plot Type API
+ * @author gnosii
+ */
+public class TownyTypeListener implements Listener {
+	
+	@EventHandler
+	public void onTownyBuild(TownyBuildEvent event) {
+		TownBlock tb = event.getTownBlock();
```

Comment:
All of these events are fired in the wilderness as well as in Towns, so you will need to check if the event is in the wilderness (and return) before you start acting upon the tb variable here.

Reply:
Resolved in commit 66f5a48